src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -929,6 +929,8 @@ def load_arguments(self, _):
             is_preview=True,
             arg_type=get_enum_type(health_probe_modes),
         )
+        # virtual machines
+        c.argument("vm_sizes", is_preview=True)
```

Comment:
If PM agrees to change the behavior of this option to use spaces to separate multiple values, may set `nargs="*"` and python/cli argument parse would help parse user input to a list of strings, which would save your work to split user input.

Reply:
don't know how others think about it, but I think the difference is not significant. If we don't accept spaces users don't need to add extra quotes around it. We may discuss it more when testing it in preview stage.